packages/core/src/exchanges/cache.ts

```diff
@@ -131,10 +140,18 @@ export const afterMutation = (
   const pendingOperations = new Set<number>();
   const { additionalTypenames } = response.operation.context;
 
-  [
+  const invalidTypeNames = [
     ...collectTypesFromResponse(response.data),
     ...(additionalTypenames || []),
-  ].forEach(typeName => {
+  ];
+
+  client.eventTarget.dispatchEvent({
+    type: 'cacheInvalidation',
+    message: `The following typenames have been invalidated: ${invalidTypeNames}`,
+    data: { value: invalidTypeNames },
```

Comment:
I think for debugging it might be useful to include the operation as part of the cache invalidation event

Reply:
Thanks sofia! Done :+1: